mobile.css

```diff
@@ -0,0 +1,199 @@
+
+.home-page {
+  background: #fff;
+  border-radius: 0 0 0 9%;
+}
+
+#navbar {
+  display: flex;
+}
+
+#navbar nav:hover {
+  cursor: pointer;
+}
+
+.menuIcon {
+  width: 45px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.internal-link {
+  display: none;
+}
+
+#home_presentation {
+  background-image:
+    url("images/Ellipse 11.png"),
+    url("images/Rectangle 4.png"),
+    url("images/Ellipse\ 3.png"),
+    url("images/Ellipse\ 8.png"),
+    url("images/Ellipse\ 5.png"),
+    url("images/Rectangle\ 7.png"),
+    url("images/Ellipse 12.png"),
+    url("images/Ellipse 4.png"),
+    url("images/Rectangle\ 5.png"),
+    url("images/Ellipse\ 13.png"),
+    url("images/Ellipse\ 15.png");
+  background-position:
+    9px 79px,
+    294px 68px,
+    323px 105px,
+    214px 110px,
+    125px 106px,
+    127px 98px,
+    370px 385px,
+    290px 509px,
+    240px 507px,
+    357px 392px,
+    106px 497px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 600px;
```

Comment:
- [ ] Kindly change the height attribute value to `height: 618px` and work on the title text font to match the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
### **Figmal Design**
![fig](https://user-images.githubusercontent.com/55596970/185603524-122f1e39-268e-45fa-b7e3-6e4536dbf70c.png)
### **Your Design**
![urs](https://user-images.githubusercontent.com/55596970/185603613-42ab22fa-b246-4968-b211-02cdb551a1a5.png)